troposphere/static/js/components/modals/instance/InstanceLaunchWizardModal.jsx

```diff
@@ -403,13 +415,35 @@ export default React.createClass({
     },
 
     onProjectCreateConfirm: function(name, description) {
-        this.viewBasic();
+        let sshKey = stores.SSHKeyStore.getAll();
+        if (sshKey) {
+            if (this.view != "IMAGE_VIEW" && sshKey.length === 0) {
+                this.view = "SSH_VIEW";
+                this.viewSSH();
```

Comment:
Right. I am trying to stay with the current code base as much as possible. But I agree with you that `setView` would be much more clear. I will add it to the to-do list and plan to do it after the summit.